### DISCUSSION TITLE
Please do something about the spam here!

### DISCUSSION THREAD

**Discussion by sketchbuch** [microsoft/vscode-discussions#1820] [Extension Development QnA] [answered]
i.e. Minecraft posts

**Comment by gjsjohnmurray** [comment]
Agreed. I am reporting the spammers to GH as fast as I can, but the reporting mechanism limits me to 2 reports before I have to wait for a timer to reset (maybe 10 minutes). See https://github.com/orgs/community/discussions/137170 about that limit, and https://github.com/orgs/community/discussions/121439#discussioncomment-10818112 for a contribution by me to another discussion about Discussions spam.

**Comment by alefragnani** [comment]
Glad to know you are reporting the spammers to GH @gjsjohnmurray , thank you! 

And I have two suggestions that could help cleaning the discussions repo:

1. Add **Actions** support to the discussion forum, with automations that allows any user to _label suspicious spam posts_, making easy to the VS Code team and/or moderators, to delete the spam posts
2. Add a few of us (the most active users), as **moderators** of the discussion forum, with the ability to _label_ spam post, or even delete them.

The Actions idea is similar to the `bot commands` that exists in the VS Code repo (https://github.com/microsoft/vscode/wiki/Issue-Tracking#inbox-tracking).

Adding  @isidorn , @olguzzar, @burkeholland  to the discussion.

What do you think?

**Comment by olguzzar** [comment, marked as answer]
Thanks so much for bringing this up - we've been doing our best to delete these as soon as we see them, but we sometimes can't act fast enough. Like @gjsjohnmurray, we've tried reporting the users as well, but GitHub's limits essentially make this impractical, and the spam just returns with new accounts anyways. Our team is trying to figure out the best way to automate this and/or prevent it altogether and will add updates to this discussion as we have them!

In the meantime, we have given @alefragnani and @gjsjohnmurray Triage permissions so that y'all can delete these posts if you see them before we can get to them. We greatly appreciate all the work y'all do in this forum (and elsewhere for VS Code!) and hope this makes it a little easier for y'all to participate!

**Reply by isidorn** [reply]
We should also continue to complain to GitHub ;) @olguzzar
